crypto/evp/pmeth_lib.c

```diff
@@ -29,6 +29,7 @@ static const EVP_PKEY_METHOD *standard_methods[] = {
 #ifndef OPENSSL_NO_DH
     &dh_pkey_meth,
 #endif
+    &pbkdf2_pkey_meth,
```

Comment:
It should probably be optional -- i.e. configurable.  Guarded by `OPENSSL_NO_PBKDF2` most likely.